Makefile

```diff
@@ -143,19 +143,6 @@ install-tools:
 	cd $(TOOLS_MOD_DIR) && go install golang.org/x/exp/cmd/apidiff
 	cd $(TOOLS_MOD_DIR) && go install golang.org/x/tools/cmd/goimports
 
-.PHONY: otelcol
-otelcol:
```

Comment:
I would prefer to keep this makefile target. I use all the time when testing locally. The github action can go.